SWLOR.Game.Server/Service/CraftService/RecipeType.cs

```diff
@@ -607,6 +607,53 @@ public enum RecipeType
         FabricationTerminal = 1203,
         Refinery = 1204,
         SmitheryBench = 1205,
+        RugClassicLightBrown = 1206,
+        PileofCushionsWhite = 1207,
+        Oven = 1208,
```

Comment:
You have two entries for 1208 here which throw off all subsequent numbers.

Reply:
note to self: corrected,  1226 left free as result of deleting wooden wall planks. will put something here later.